<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Utils::LoginWidget</class>
 <widget class="QWidget" name="Utils::LoginWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>78</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="loginLabel">
     <property name="text">
      <string>&amp;Login</string>
     </property>
     <property name="buddy">
      <cstring>login</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="Utils::LineEditEchoSwitcher" name="login"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="passwordLabel">
     <property name="text">
      <string>&amp;Password</string>
     </property>
     <property name="buddy">
      <cstring>password</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="Utils::LineEditEchoSwitcher" name="password"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::LineEditEchoSwitcher</class>
   <extends>QLineEdit</extends>
   <header>utils/widgets/lineeditechoswitcher.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
